xcodeproj/internal/target.bzl

```diff
@@ -922,7 +921,7 @@ def _skip_target(*, target, deps, transitive_infos):
             transitive_infos = transitive_infos,
         ),
         inputs = input_files.merge(
-            attrs_info = target[InputFileAttributesInfo],
+            attrs_info = None,
```

Comment:
It might be helpful to add the comment from the PR to the code.

Reply:
I plan to completely revamp input collection "soon" which does away with most of this, so I think it's fine to go without a comment for now.